__tests__/app.test.js

```diff
@@ -356,4 +441,34 @@ describe("my express app", () => {
       });
     });
   });
+  describe("/api/comments/:comment_id", () => {
+    describe("DELETE /api/comments/:comment_id", () => {
+      test("204: responds with 204 status and no content", () => {
+        return request(app)
+          .delete("/api/comments/2")
+          .expect(204)
+          .then(({ body }) => {

```

Comment:
So I don't think this `.then` is still necessary, because DELETE requests essentially don't have return values, but what I would do is make a query to the database requesting this specific comment just to check that it doesn't exist anymore!